src/Kernel.php

```diff
@@ -102,6 +102,18 @@ public function registerBundles(): iterable
             }
         }
 
+        $additionalBundlesPath = $this->getAdditionalBundlesPath();
```

Comment:
The file should take precedence over the old string list.
If the file is there, load it and ignore the env; maybe w/ a note?